internal/cache/file/downloader/jm_parallel_downloads_test.go

```diff
@@ -98,31 +97,25 @@ func TestParallelDownloads(t *testing.T) {
 		parallelDownloadsPerFile int64
 		maxParallelDownloads     int64
 		downloadOffset           int64
-		expectedOffset           int64
 		subscribedOffset         int64
 	}{
 		{
 			name:                     "download in chunks of concurrency * readReqSize",
 			objectSize:               15 * util.MiB,
 			readReqSize:              3,
-			parallelDownloadsPerFile: math.MaxInt,
+			parallelDownloadsPerFile: 100,
```

Comment:
nit: change the name "download in chunks of concurrency * readReqSize" of this unit test?